R/to_alias.R

```diff
@@ -206,19 +206,41 @@ from_name_to_alias <- function() {
     "8",           "eight",
     "9",            "nine",
     "aktg",              "ag",
+    "aktiengesellschaft", "ag",
```

Comment:
You may point Anne to the diff of R/to_alias.R on this PR. That way she can see exactly the new rows you added.

Reply:
@lindadelacombaz could you explain, why you change the "8" to "eight"? As this was not in [the sheet ](https://docs.google.com/spreadsheets/d/1UMrl3pNxbqEPlGfO-WO1_NKGUXYSRn8lHGr0YdwmD_8/edit#gid=0) I created.

Reply:
@lindadelacombaz ah I think I understood - the "8" and "eight" is from PACTA right? Then all good :)

Reply:
@AnneSchoenauer yes, exactly. I did not change nor add these lines of code, only the green highlighted ones :)